venue/tasks.py

```diff
@@ -313,7 +347,7 @@ def update_data(forum_profile_id=None):
     for profile in forum_profiles:
         subtasks.append(scrape_forum_profile.s(profile.id))
 
-    chord(subtasks)(compute_points.s())
+    chord(subtasks)(compute_points.si()).apply_async()
```

Comment:
Are you sure that `chord(subtasks)(compute_points.s())` doesn't execute tasks? It worked well during the tests.

Reply:
On second thought, the `apply_async()` is not necessary because we already execute `update_data()` itself in async mode. This is not a breaking change, the chord will still execute but it will run on another worker process. But I'll remove it, it's not necessary. `chord(subtasks)(compute_points.si())` should be enough.

If you noticed, I used `.si()` instead of `.s()` for the callback, so that the results from the subtasks will not be passed as arguments to callback.